source/e_2d_flood_tutorial.rst

```diff
@@ -0,0 +1,399 @@
+
+#) Right-click to stop drawing the Exchange line. A popup screen with the Feature Attributes should now appear.
+
+#) Fill in the following parameters in the *channel* tab:
+
+    * ID: filled in automatically
+    * Code: give a name you can identify it by
+    * Exchange level [m]: leave empty
+    * Channel ID: ID of the channel outside of the DEM (2)
+
+#) Click *OK* and click the *Toggle editing mode* button in the toolbar and save your edits to this layer.
+
+#) You can add a potential breach in the same way as you did in :ref:`adding_potential_breach`. Make sure to snap the start of the potential breach to the channel and let it end on the DEM behind the exchange line.
+
+.. VRAAG: klopt dit zo?
+
+Uploading a revision
+----------------------
+The next step is to check the schematisation, upload its as a second :ref:`revision` and process it into a :ref:`threedimodel`. 
+
+#) In the 3Di Schematisation Editor toolbar, click *Save to Spatialite* (|save_to_spatialite|). Wait for this process to finish.
+
+#) Click on the 3Di Models and Simulations icon (|modelsSimulations|). You should now see the 3Di Models and Simulations panel.
+
+#) Click the upload button (|upload|) in the 3Di Models and Simulations panel.
+
+#) In the dialog that has appeared, click *New upload* and click *Next*.
+
+#) Click *Check schematisation*. This will check your schematisations for any errors that make it impossible to generate a valid 3Di model and simulation template.
+
+#) Continue to the next screen. Here you have to fill in a commit message that describes the changes your have made relative to the previous revision. For example: "Added channel with potential breach".
+
+#) Click *Start upload*. Check if it is the upload is successful and if the uploaded data is successfully processed into a 3Di model.  
+
+Your 3Di model is now ready for simulation! The model is now also available on `3Di Live <https://www.3di.live/>`_ and the `management screens <https://management.3di.live>`_.
+
+
+Running a simulation 
+----------------------
+
+You will now start a simulation with the 3Di model you have created in the 3Di Modeller Interface: 
+
+#) In the 3Di Models and Simulations panel, click *Simulate* (|simulate|) > *New simulation*.  
+
+#) Select your model and simulation template and click *Next*. A new dialog opens with several options for your simulation.  
+
+#) Check the box for *Include breaches* (keep *Include initial conditions* and * Include boundary conditions* checked). Click *Next*.
+
+#) Give your simulation a name. Click *Next*.
+
+#) Set the duration of your simulation to 4 hours. Click *Next*.
+
+#) Accept the Boundary conditions as they are by clicking *Next*.
+
+#) Accept the Initial conditions as they are by clicking *Next*.
+
+#) Fill in the following parameters for Breaches and then click *Next*.
+
+    * ID of breach: 1 (if your model only contains 1 breach)
+    * Duration till max depth: 0.100 hours
```

Comment:
Is it 'till' of 'until'. until is the formal form and should be used (if that's not the case, I'd open a ticket).

Reply:
er moet een ticket worden aangemaakt